authors/apps/articles/renderers.py

```diff
@@ -53,3 +53,24 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
                 'article': data
             }
         })
+
+
+class RatingJSONRenderer(JSONRenderer):
```

Comment:
You can use this class instead https://github.com/andela/ah-code-blooded/blob/develop/authors/apps/core/renderers.py